ObjectCreation/G3d.FCMacro

```diff
@@ -0,0 +1,475 @@
+        "Length",
+        "Total length of the lateral extrusion along the X axis (it must be always larger than the lateral offset)",
+        "Offset",
+        "Offset from the sketch plane from which the extrusion along the Z axis will start",
+        "Length",
+        "Total length of the vertical extrusion along the Z axis (it must be always larger than the vertical offset)",
+        "3D-Part",
+        "3D-Part_Views"
+    ],
+    [ ## Stringhe per la traduzione in qualsiasi lingua (stringhe in italiano)
+
+        "Generatore 3D",
+        "Seleziona uno schizzo in 2 o 3 viste:",
+        "Vista frontale - (Fronte/Retro):",
+        "Spunta questa casella se desideri utilizzare la vista frontale ed assegnarle uno schizzo",
+        "Vista laterale - (Destra/Sinistra):",
+        "Spunta questa casella se desideri utilizzare la vista laterale ed assegnarle uno schizzo",
+        "Vista in pianta - (Alto/Basso):",
+        "Spunta questa casella se desideri utilizzare la vista in pianta ed assegnarle uno schizzo",
+        "Tol. 3D (mm.):",
+        "Lunghezza in mm di sovraestrusione. Non dovrebbe influenzare l'oggetto finale. Se non si necessita questa tolleranza impostare il valore relativo a 0 mm",
+        "Crea un nuovo gruppo (cartella) con le viste (sketch)",
+        "Nasconde le viste quando viene generato il solido",
+        "Fusione (intersezione)",
+        "Versione:",
+        "Breve guida all'uso dello script:",
+        "Crea uno sketch per ogni vista necessaria a generare il solido. Sono necessari almeno due sketch.",
+        "Quindi clicca sull'icona dello script e seleziona uno schizzo per ciascuna vista necessaria.",
+        "Infine, premi il pulsante OK per generare il solido.",
+        "Documentazione:",
+        "Offset",
+        "Distanza rispetto al piano dello sketch frontale da cui inizierà l'estrusione lungo l'asse Y (spostamento in Y)",
+        "Lunghezza",
+        "Lunghezza totale dell'estrusione frontale lungo l'asse Y (dovrà essere sempre più grande dell'offset frontale)",
+        "Offset",
+        "Distanza rispetto al piano dello sketch laterale da cui inizierà l'estrusione lungo l'asse X (spostamento in X)",
+        "Lunghezza",
+        "Lunghezza totale dell'estrusione laterale lungo l'asse X (dovrà essere sempre più grande dell'offset laterale)",
+        "Offset",
+        "Distanza rispetto al piano dello sketch in pianta da cui inizierà l'estrusione lungo l'asse Z (spostamento in Z)",
+        "Lunghezza",
+        "Lunghezza totale dell'estrusione verticale lungo l'asse Z (dovrà essere sempre più grande dell'offset in pianta)",
+        "Pezzo3D",
+        "Pezzo3D_Viste"
+    ]
+        ]
+
+
+
+rutaMacro = FreeCAD.ParamGet("User parameter:BaseApp/Preferences/Macro")     
+rutaUI = rutaMacro.GetString("MacroPath","") + "/g3d/ui/g3d15.ui"
+rutaUI = rutaUI.replace("\\","/")
+
+minimoExtrusion = 0.01
```

Comment:
Could you use `App.Units.Quantity('0.01 mm')`?

Reply:
Done